beacon-chain/sync/decode_pubsub.go

```diff
@@ -43,6 +47,13 @@ func (s *Service) decodePubsubMessage(msg *pubsub.Message) (ssz.Unmarshaler, err
 	if !ok {
 		return nil, errors.Errorf("message of %T does not support marshaller interface", base)
 	}
+	// Handle different message types across forks.
+	if topic == p2p.BlockSubnetTopicFormat {
```

Comment:
How does this equality work? BlockSubnetTopicFormat is `/eth/%x/beacon_block`, but wouldn't topic include the digest?

Reply:
Ah, nevermind, we replace the topic digest with the formatting.